guides/advanced_guides/faceted_search.md

```diff
@@ -0,0 +1,49 @@
+# Faceted Search
+
+Faceted search is a feature provided out-of-the-box by MeiliSearch. Faceting allows to classify search results into categories that are called **facets**.
+
+> For a movie, its title, director, and its release date can be used as facets.
+
+The user can then narrow down search results by selecting facets, which is similar to applying filters to refine the results.
+
+Both faceting and filtering help narrow down the search results. However, **faceting differs from [filtering](/guides/advanced_guides/filtering.md)**.
+
+Faceted attributes **must be declared at indexing time**.
+
+## Using Facets
+
+In order to implement the all new facets, we at meilisearch have added a new attribute `attributesForFaceting`. With the default value of `Null` it accepts `[String]`.
+It should be noted that passing an empty array will recet all faceting attribute in other words any call to settings with a value for the attribute attributesForFaceting overwrites the currently set faceting attributes.
+
+A call to [POST]/indexes/:index_uid/settings should list the currently set facets.
```

Comment:
As a general rule of thumb, in a spec, thing "should" do something, but in  documentation, things "do" things :)

```suggestion
A call to [POST]/indexes/:index_uid/settings lists the currently set facets.
```